go/porcelain/deploy.go

```diff
@@ -327,8 +329,16 @@ func (n *Netlify) DoDeploy(ctx context.Context, options *DeployOptions, deploy *
 		return deploy, nil
 	}
 
-	if err := n.uploadFiles(ctx, deploy, options.files, options.Observer, fileUpload, options.UploadTimeout); err != nil {
-		return nil, err
+	// Temporary way to ensure safe rollout of skip uploading files with same digest optimization
+	// We'll
```

Comment:
You had more to say 😅 or this is the whole comment?

Reply:
Fixed in - https://github.com/netlify/open-api/pull/425/commits/552dc0d5e54296b399ff1dfa6093d17d9ee84ab6